pydantic/schema.py

```diff
@@ -0,0 +1,392 @@
+from datetime import date, datetime, time, timedelta
+from decimal import Decimal
+from enum import Enum
+from pathlib import Path
+from typing import Any, Dict, Sequence, Set, Tuple, Type
+from uuid import UUID
+
+from . import main
+from .fields import Field, Shape
+from .json import pydantic_encoder
+from .types import DSN, UUID1, UUID3, UUID4, UUID5, DirectoryPath, EmailStr, FilePath, Json, NameEmail, UrlStr
+from .utils import clean_docstring
+
+
```

Comment:
best to put all these "helper" methods at the bottom of the file and the "primary" methods (those people might want to read first) at the top.

Also, you should probably implement `__all__` here.

Reply:
Done